### DISCUSSION TITLE
InputTagControl Issue

### DISCUSSION THREAD

**Discussion by mat-guthrie-loupa** [havit/Havit.Blazor#448] [Q&A]
Hi,

I'm trying to set up the InputTags control but when entered the tags appear outside and above the text box. What am I missing? 

![image](https://user-images.githubusercontent.com/104560023/211101097-09df7659-bc46-44c5-9f65-22faf6aa6b28.png)

Thanks

Mat

**Comment by hakenr** [comment]
@mat-guthrie-loupa Hi, this seem you have some troubles with CSS definitions.
Make sure you are referencing Bootstrap 5.2 CSS files (or our Bootstrap build).
If you think your CSS is correct, can you provide me with a simple repro?
